src/test/resources/api/dummy_restipiexample/_put/rqput.json

```diff
@@ -0,0 +1 @@
+{
  "data": {
    "name": "test",
    "salary": "123",
    "age": "23"
  }
}
```

Comment:
does your tests depend on specific user? what will happen if this test user doesn't exist? tests should be atomic